backend/config/curation-api.yml

```
@@ -356,7 +356,37 @@ paths:
           $ref: "#/components/responses/401"
         "404":
           $ref: "#/components/responses/404"
-
+  /v1/checks/submission_key:
+    put:
+      summary: Verify the result of submitting a dataset
```

> Verify

☝️ 👍

---

```suggestion
      summary: Verify the result of submitting a dataset: 'create' vs 'update'
```

---

```suggestion
      summary: Quickly check the result of submitting a dataset.
```

---

```suggestion
      summary: Determine whether a Dataset submission under the given `collection_id` and identifier (`curator_tag` or `dataset_id`) will produce a new Dataset or update an existing Dataset.
```

This, assuming that we change the query to take a `collection_id` and a `curator_tag` or `dataset_id`.